addons/crm/models/crm_lead.py

```diff
@@ -428,6 +370,11 @@ def write(self, vals):
             self._update_probability()
 
         return write_result
+    
+    def toggle_active(self):
+        for lead in self:
+            lead._compute_probabilities()
+        super(Lead, self).toggle_active()
```

Comment:
Not exactly, we should only recompute if active is True after super.

Reply:
It should have looked at toggle_active in models, I thought it always put active to true. I corrected it, sorry.